x-pack/plugins/security_solution/public/timelines/components/timeline/tabs_content/index.tsx

```diff
@@ -250,14 +262,18 @@ const ActiveTimelineTab = memo<ActiveTimelineTabProps>(
               overflow: hidden !important;
             `}
           >
-            <AssistantTab
-              renderCellValue={renderCellValue}
-              rowRenderers={rowRenderers}
-              timelineId={timelineId}
-              shouldRefocusPrompt={
-                showTimeline && activeTimelineTab === TimelineTabs.securityAssistant
-              }
-            />
+            {(activeTimelineTab === TimelineTabs.securityAssistant ||
```

Comment:
The `activeTimelineTab === TimelineTabs.securityAssistant` condition (correctly) prevents Timeline's instance of the assistant from being rendered until the `Security assistant` tab is focused for the first time